plugins/woocommerce/client/admin/client/order-attribution-install-banner/use-order-attribution-install-banner.js

```diff
@@ -2,17 +2,67 @@
  * External dependencies
  */
 import { useDispatch, useSelect } from '@wordpress/data';
+import { useCallback, useMemo } from '@wordpress/element';
 import {
 	OPTIONS_STORE_NAME,
 	PLUGINS_STORE_NAME,
 	useUser,
 } from '@woocommerce/data';
 import { recordEvent } from '@woocommerce/tracks';
 import { getPath } from '@woocommerce/navigation';
+import { isWcVersion } from '@woocommerce/settings';
+
+/**
+ * Internal dependencies
+ */
+import { STORE_KEY } from '~/marketing/data/constants';
+import '~/marketing/data';
 
 const OPTION_NAME_BANNER_DISMISSED =
 	'woocommerce_order_attribution_install_banner_dismissed';
 const OPTION_VALUE_YES = 'yes';
+const OPTION_NAME_REMOTE_VARIANT_ASSIGNMENT =
+	'woocommerce_remote_variant_assignment';
+
+const getThreshold = ( percentages ) => {
+	const defaultPercentages = [
+		[ '9.7', 100 ], // 100%
+		[ '9.6', 60 ], // 60%
+		[ '9.5', 10 ], // 10%
+	];
+
+	if ( ! Array.isArray( percentages ) || percentages.length === 0 ) {
+		percentages = defaultPercentages;
+	}
+
+	for ( let [ version, percentage ] of percentages ) {
+		if ( isWcVersion( version, '>=' ) ) {
```

Comment:
Let's consider we are in WC v9.7.

If for some reason v9.5 comes before v9.7 in the API  (let's suppose someone updates the recommendations.json wrong) then 10% will be returned as v9.5 satisfies  `isWcVersion( version, '>=' )`. However, we expect 100% because we are on v9.7

Reply:
Agreed with that, it would probably be safer to sort the percentage values before using it.

Reply:
I mean, sort by the version 😄